.travis.yml

```diff
@@ -97,6 +97,9 @@ matrix:
       rust: beta
 
   allow_failures:
+    # There are too many inconsistent failures on this arch to make it Tier 1.
```

Comment:
Frankly, I would rather just add `DISABLE_TESTS=1` to powerpc.  I don't think we're likely to learn much by running them, especially since we can't trust any of their failures.